src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/IReferenceCountedDisposable.cs

```diff
@@ -13,6 +13,9 @@ namespace Roslyn.Utilities
     /// </summary>
     /// <typeparam name="T"></typeparam>
     internal interface IReferenceCountedDisposable<out T> : IDisposable
+#if !CODE_STYLE
+        , IAsyncDisposable
```

Comment:
not sure why CODE_STYLE doesn't have access to this type.  seems odd to me.

Reply:
It's defined in Microsoft.Bcl.AsyncInterfaces. We would have to ship a new assembly in the CodeStyle layer to use this type while still targeting netstandard2.0.